feature-toggles/manage/manageTogglesController.js

```diff
@@ -0,0 +1,46 @@
+(function() {
+    'use strict';
+
+    const app = angular.module('app');
+
+    app.controller('ManageTogglesController', function(ManageTogglesService, AuthService, MessageService) {
+        const manageTogglesCtrl = this;
+        manageTogglesCtrl.isLoading = false;
+        manageTogglesCtrl.features = [];
+        manageTogglesCtrl.modifiedFeatures = [];
+
+        manageTogglesCtrl.logout = function() {
+            AuthService.logout();
+        };
+
+        manageTogglesCtrl.addModifiedFeature = function(feature) {
```

Comment:
inst better to mark feature as changed, and than only on save you filter the features marked?
every time each of then is changed, you do an _.find, so you are going throw the list n times instead of just one.

Also you are only adding the feature without verifying if it has really changed.
For example: I change from admin to test; so this feature is added. but i changed my mind and don't want to modify this feature anymore, before hit `save`, I change it back again to admin. So technically I didn't modify the feature, but it already is at the `modifiedFeatures` array, and it will be saved.
So, if you send the complete array to save without doing _.find it wont make too much difference, unless you verify which of them was really modified.